draft-barnes-acme.md

```diff
@@ -1676,19 +1679,17 @@ validation (required, string):
 Given a Challenge/Response pair, the ACME server verifies the client's control
 of the domain by verifying that the TLS server was configured appropriately.
 
-1. Verify the validation JWS using the account key for which the challenge was
-   issued
+1. Verify that the validation JWS uses the account key for which the challenge
+   was issued
```

Comment:
This change is incorrect.  If you're going to have this step, then you need to have a separate step where you check the signature on the JWS (i.e., "verify" the signature).